bench/setup_noop_bench_docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+trap "exit" INT TERM
+trap "kill 0" EXIT
+
+export CLIPPER_MODEL_NAME="bench_noop"
+export CLIPPER_MODEL_VERSION="1"
+export CLIPPER_MODEL_PATH="model/"
+
+# Sets CLIPPER_IP to AWS instance's IP.
+# This will only work if the docker container corresponding to noop_container
+# is running on the same host as the clipper/query_frontend container.
+export CLIPPER_IP=$(curl http://169.254.169.254/latest/meta-data/local-ipv4)
```

Comment:
This docker script is AWS-specific, and we should make note of that in the README as well as provide an alternative option to build a container with a custom IP.